tools/android_sdk/create_cipd_packages.sh

```diff
@@ -26,13 +27,13 @@ print_usage () {
   echo "and should only be run on linux or macos hosts."
 }
 
-# Validate version is provided
+# Validate version or argument is provided.
```

Comment:
The following code would read better if you copied $1 into a named variable:

VERSION_TAG=$1